cubic-server/Entity.cpp

```diff
@@ -1,5 +1,27 @@
 #include "Entity.hpp"
 
+Entity::Entity()
+{
+    _onFire = false;
+    _crouching = false;
+    _sprinting = false;
+    _swimming = false;
+    _invisible = false;
+    _glowing = false;
+    _flyingWithElytra = false;
+    _airTicks = 300;
+    _customName = "";
+    _customNameVisible = false;
+    _silent = false;
+    _noGravity = false;
+    _pose = Pose::STANDING;
+    _tickFrozenInPowderedSnow = 0;
+}
+
+Entity::~Entity()
```

Comment:
The destructor has been deleted

Reply:
I know but if I remove that the linker cannot link for a reason I don't understand
![image](https://user-images.githubusercontent.com/60519602/208809603-56cc36d7-cc5e-40af-858f-50e637d2b43b.png)

Reply:
https://stackoverflow.com/a/1219618

> Note: The destructor is the only method that even if it is pure virtual has to have an implementation in order to instantiate derived classes (yes pure virtual functions can have implementations).